finders/metadata/esi-funds.json

```diff
@@ -9,6 +9,11 @@
   "filter": {
     "document_type": "european_structural_investment_fund"
   },
-  "default_order": "-closing_date",
```

Comment:
Amended now! Local error was because I needed to update govuk-content-schemas and rummager for the "-closing_date" ordering to be recognised.